### DISCUSSION TITLE
Tracking OpenRadioss in downstream packaging repositories

### DISCUSSION THREAD

**Discussion by luzpaz** [OpenRadioss/OpenRadioss#3911] [Ideas]
Currently Repology[^1] is showing this:  

<a href="https://repology.org/project/openradioss-engine/versions">
    <img src="https://repology.org/badge/vertical-allrepos/openradioss-engine.svg?columns=3&header=openradioss-engine&exclude_unsupported=1" alt="Packaging status" align="right">
</a><a href="https://repology.org/project/openradioss-starter/versions">
    <img src="https://repology.org/badge/vertical-allrepos/openradioss-starter.svg?columns=3&header=openradioss-starter&exclude_unsupported=1" alt="Packaging status" align="right">
</a>

It would be beneficial to get OpenRadioss into the AUR, nixpkg, and other popular repos. 

[^1]: https://repology.org

**Comment by elequiniou** [comment]
Thanks @luzpaz, as it is not a bug that requires a fix, but more an enhancement to be investigated, I propose to move to discussion forum, the time someone knowledgeable in such package address this request. Thanks a lot!
